src/DynamoApplications/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Reflection;
```

Comment:
Ensure that your new projects are using `AssemblySharedInfo.cs`. This will ensure correct version numbering.